src/main/java/com/statisticalcalculator/CalcDTO/SaveCalcDTO.java

```diff
@@ -0,0 +1,81 @@
+package com.statisticalcalculator.CalcDTO;
+
+public class SaveCalcDTO {
+	
+	private int calcID;
+	private String date;
+	private String setNumbers;
+	private int count;
+	private int maxVal;
+	private int minVal;

```

Comment:
Avoid the urge to abbreviate names.  The time you save in typing fewer letters will be spent many times over in trying to remember what abbreviation you used.